src/west/manifest-schema.yml

```diff
@@ -9,54 +9,32 @@
 ##
 ## However, the features don't map 1:1.
 
-# The top-level manifest is a map. There may be multiple sections in the
-# manifest file. Each section can be validated by their own schema.
-# This schema validates the 'manifest' section.
+# --------------------------------------------------------------
+#
+#  DO NOT CHANGE THIS FILE WITHOUT UPDATING THE DOCUMENTATION!
```

Comment:
The same comments given to: 
`src/west/commands/west-commands-schema.yml`
also applies to this schema file.